pkg/storage/export/export/export.go

```diff
@@ -877,8 +876,8 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 	podManifest.Labels = map[string]string{exportServiceLabel: vmExport.Name}
 	podManifest.Annotations = map[string]string{annCertParams: scp}
 	podManifest.Spec.SecurityContext = &corev1.PodSecurityContext{
-		RunAsNonRoot:   pointer.Bool(true),
-		FSGroup:        pointer.Int64Ptr(kvm),
+		RunAsNonRoot:   pointer.P(true),
+		FSGroup:        pointer.P(int64(kvm)),
```

Comment:
nit: AFAICT this is the only usage of `kvm`, maybe it's better to change it to `int64`?

Reply:
@iholder101 yes, used in only one place. Now, what to do ?. shall i open for new comers?

Reply:
It's just a nit (=not important), I don't think it's crucial in any sense :)

Reply:
ok